src/site/twitch.tv/modules/chat/components/user/UserCard.vue

```diff
@@ -32,6 +32,19 @@
 						<p v-if="data.targetUser.relationship.followedAt">
 							{{ t("user_card.following_since_date", { date: data.targetUser.relationship.followedAt }) }}
 						</p>
+						<p v-if="data.targetUser.relationship.subscription.months">
+							{{
```

Comment:
For the sake of readability consider using a `v-else` instead

Reply:
Hope it's more clear now. Also rebased on the most recent master branch

Reply:
I added also the creation date to the user card. See top comment. If not in scope I can revert the last commit.